src/quo2/components/profile/collectible/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.profile.collectible.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.profile.collectible.style :as style]
+            [react-native.core :as rn]))
+
+(defn plus-n-tile
+  [n]
+  [rn/view {:style (merge style/bottom-right (style/plus-n))}
+   [text/text
+    {:style  (style/plus-n-text)
+     :size   :paragraph-2
+     :weight :medium}
+    (str "+" n)]])
+
+(defn tile
+  [{:keys [uri size]}]
+  [rn/image
+   {:style  (style/tile-style-by-size size)
+    :source {:uri uri}}])
+
+(defn two-tiles
+  [{:keys [imgs size]}]
+  [:<>
+   [rn/view {:style style/top-left}
+    [tile {:uri (first imgs) :size size}]]
+   [rn/view {:style style/bottom-right}
+    [tile {:uri (second imgs) :size size}]]])
+
+(defn three-tiles
+  [{:keys [tiles size]}]
+  (let [[img-1 img-2 img-3 & _] tiles]
+    [:<>
+     [rn/view {:style style/top-left}
```

Comment:
we could pass style as parameter to tile, so we avoid using containers views

Reply:
Got it! it's in fact a lot better, thanks!

It has been done in `110608d493491317bedc25bfa700609323fc0164`